docs/policy/repo-layout-migration-2025.md

```diff
@@ -0,0 +1,45 @@
+OSG Software Repo Layout Migration
+==================================
+
+11 December 2024
+
+On Monday, January 6th, 2025, the OSG Software Team will be upgrading the server that hosts the OSG Software repos
+(<https://repo.osg-htc.org> a.k.a. <https://repo.opensciencegrid.org>),
+which will result in changes to the directory layout for the OSG 23 and OSG 24 YUM repos.
+
+Users should not be affected since the URLs of the repos themselves will not change.
```

Comment:
```suggestion
Users installing packages should not be affected since the URLs of the repositories themselves will not change.
```

We have so many different kinds of users